katello.gemspec

```diff
@@ -48,6 +48,7 @@ Gem::Specification.new do |gem|
   gem.add_development_dependency "mocha"
   gem.add_development_dependency "vcr", "< 4.0.0"
   gem.add_development_dependency "webmock"
+  gem.add_development_dependency "rubocop", "~> 0.49.0"
```

Comment:
@ehelms Hmm.. is this not the way to do this? I tested rubocop locally, and this resolves to 0.49.1 which passes.

Reply:
Oh this is a CI configuration issue due to how I changed to use Github pull request builder. You don't actually need this commit given 1.16-stable locks to 0.49.0